README.md

```diff
@@ -14,14 +14,34 @@
 
 ## Live Demo
 
-[Live Demo Link]()
+[Live Demo Link](https://nyame-wolf.github.io/Leaderboard/)
 
 ## Project Milestones
 
 - **Milestone 0 - project setup (0.5h)** ✅
 - **Milestone 1 - Leaderboard: hit the API (2.5h)**✅
 - **Milestone 2 - Leaderboard: final touches (4h)**
```

Comment:
- Please, kindly remove this added milestone as this is not required you add it, instead rename the "Project Milestones" to "Getting Started" as this is the correct header that should have all you've added in that section to make your readme more professional following the microverse readme template.